src/VisualStudio/Core/Def/ValueTracking/ValueTrackingToolWindow.cs

```diff
@@ -13,29 +13,33 @@ namespace Microsoft.VisualStudio.LanguageServices.ValueTracking
     internal class ValueTrackingToolWindow : ToolWindowPane
     {
         public static ValueTrackingToolWindow? Instance { get; set; }
-        private readonly ValueTrackingTreeViewModel _viewModel;
+        public ValueTrackingTreeViewModel? ViewModel { get; }
 
-        public ValueTrackingToolWindow(ValueTrackingTreeItemViewModel root)
-            : base(null)
+        public ValueTrackingToolWindow() : base(null)
```

Comment:
what is this contructor for?

Reply:
If a user has the tool window open already, this is the constructor that gets called